### PR TITLE
Fix build for zig 0.6.0

### PR DESCRIPTION
**Pull request by lluchs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,6 +258,18 @@ pub fn main() !void {
...
+```
+
+
+            clap.parseParam("    --value <N>  Output version information and exit.") catch unreachable,
+        },
+    );
+}
+
+```
+
```
Comment (lluchs) [inline README.md:272]: This doesn't seem right, not sure where it's coming from.

File: clap.zig [hunk 2]
```diff
@@ -60,7 +60,8 @@ pub fn parseParam(line: []const u8) !Param(Help) {
     var res = Param(Help){
         .id = Help{
             .msg = line[0..0],
-            .value = line[0..0],
+            // see zig issue #4771
+            .value = &[0]u8{},
```
Comment (lluchs) [inline clap.zig:64]: The tests below fail otherwise when comparing the pointer value, see https://github.com/ziglang/zig/issues/4771

**Comment by Hejsil** [comment]
I'm sorry to say this, but I think you're looking for the [`zig-master`](https://github.com/Hejsil/zig-clap/tree/zig-master). Maybe I need to mention this branch in the README so that effort like this isn't waisted.

**Comment by Hejsil** [comment]
I'll personally merge this into `master` once my package manager gets `zig 0.6.0`

**Comment by Hejsil** [comment]
`zig-master` has been merged into `master`, so master should now compile and work with `zig 0.6.0`.

**Pull request closed without merging**